app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt

```diff
@@ -262,97 +322,127 @@ fun DownloadsScreen(navigator: DestinationsNavigator) = composing(navigator) {
                         tonalElevation = 1.dp,
                         shadowElevation = 1.dp,
                         headlineContent = {
-                            Text("$defaultName [${downloadsCount.getOrDefault(null, 0)}]")
+                            Text("$emptyLabelName [${downloadsCount.getOrDefault(null, 0)}]")
                         },
                     )
                 }
-                itemsIndexed(labelList, key = { _, item -> item.id!! }) { index, (item, _, id) ->
-                    // Not using rememberSwipeToDismissBoxState to prevent LazyColumn from reusing it
-                    val dismissState = remember { SwipeToDismissBoxState(SwipeToDismissBoxValue.Settled, density, positionalThreshold = positionalThreshold) }
-                    LaunchedEffect(dismissState) {
-                        snapshotFlow { dismissState.currentValue }.collect {
-                            if (it == SwipeToDismissBoxValue.EndToStart) {
-                                runCatching {
-                                    awaitPermissionOrCancel(confirmText = R.string.delete) {
-                                        Text(text = stringResource(R.string.delete_label, item))
-                                    }
-                                }.onSuccess {
-                                    DownloadManager.deleteLabel(item)
-                                    when (filterState.label) {
-                                        item -> switchLabel("")
-                                        null -> invalidateKey = !invalidateKey
+
+                itemsIndexed(groupLabelList, key = { _, (id, _) -> id }) { index, (id, item) ->
+                    val dismissState = when (filterMode) {
```

Comment:
Use `gesturesEnabled` parameter of `SwipeToDismissBox` instead.

Reply:
I was completely unaware of these issues..
I will make every effort to avoid it